tests/integrationv2/test_buffered_send.py

```diff
@@ -40,24 +40,21 @@ def test_SEND_BUFFER_SIZE_MIN_is_s2ns_min_buffer_size(managed_process):
     port = next(available_ports)
 
     s2n_options = ProviderOptions(mode=Provider.ServerMode,
-        port=port,
-        data_to_send="test",
-        extra_flags=['--buffered-send', SEND_BUFFER_SIZE_MIN-1])
+                                  port=port,
+                                  data_to_send="test",
+                                  extra_flags=['--buffered-send', SEND_BUFFER_SIZE_MIN])
```

Comment:
Is there a reason to still set a send buffer on the server?

Reply:
This is so something happens in the event that a connection does occur. Otherwise I'm worried that there will be a failure due to a lack of data. Maybe that's not a well founded fear.